### PR TITLE
fix transformToken

### PR DESCRIPTION
**Pull request by frankleaf**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/jindo/transform.go [hunk 1]
```diff
@@ -581,7 +581,7 @@ func (e *JindoEngine) transformSecret(runtime *datav1alpha1.JindoRuntime, value
 	return nil
 }
 
-func (e *JindoEngine) transformToken(runtime *datav1alpha1.JindoRuntime, value *Jindo) (err error) {
+func (e *JindoEngine) transformToken(value *Jindo) (err error) {
 	if len(runtime.Spec.Secret) != 0 {
```
Comment (cheyang) [inline pkg/ddc/jindo/transform.go:585]: It's failed to compile.

File: pkg/ddc/jindofsx/transform.go [hunk 2]
```diff
@@ -960,7 +960,7 @@ func (e *JindoFSxEngine) transformSecret(runtime *datav1alpha1.JindoRuntime, val
 	}
 }
 
-func (e *JindoFSxEngine) transformToken(runtime *datav1alpha1.JindoRuntime, value *Jindo) {
+func (e *JindoFSxEngine) transformToken(value *Jindo) {
```
Comment (cheyang) [inline pkg/ddc/jindofsx/transform.go:963]: Do we still need this function?

Comment (frankleaf) [reply pkg/ddc/jindofsx/transform.go:963]: of course need

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**